File: Python/lt_01_estseq/lt01_estseq_15.py
from math import pow, sqrt

cat1=(pow((float(input("Insira o valor do Cateto Oposto: "))),2))
cat2=(pow((float(input("Insira o valor do Cateto Adjacente: "))),2))
print("O valor da hipotenusa é",round((sqrt(cat1+cat2)),2),".")

File: C/lt_01_estseq/lt01_estseq_15.c
// Nome do Programador:
// Descrição:

#include <math.h>
#include <stdio.h>

int main() {

  float adj, opo, hip;

  printf("Insira o valor do Cateto Adjaente: ");
  scanf("%f", &adj);

  printf("Insira o valor do Cateto Oposto: ");
  scanf("%f", &opo);

  opo = pow(opo, 2);
  adj = pow(adj, 2);
  hip = opo + adj;
  hip = sqrtf(hip);

  printf("O valor da Hipotenusa e %.2f.\n", hip);
  return 0;
}
